README.md

```diff
@@ -1,21 +1,28 @@
-### VSTest
+# VSTest
+
 The Visual Studio Test Platform is an open and extensible test platform that enables running tests, collect diagnostics data and report results. The Test Platform supports running tests written in various test frameworks, and using a pluggable adapter model. Based on user-choice, the desired test framework and its corresponding adapter can be acquired as a vsix or as NuGet package as the case may be. Adapters can be written in terms of a public API exposed by the Test Platform.
 
 The Test Platform currently ships as part Visual Studio 2019, and in the .NET Core Tools Preview 3.
 
-### Build status
+## Build status
+
 [![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/Microsoft/vstest/microsoft.vstest.ci?branchName=main)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=935&branchName=main)
 
-### Contributing
+## Contributing
+
 There are many ways to contribute to VSTest
+
 - [Submit issues](https://github.com/Microsoft/vstest/issues) and help verify fixes as they are checked in.
 - Review the [source code changes](https://github.com/Microsoft/vstest/pulls).
 - [Contribute features and fixes](https://github.com/Microsoft/vstest-docs/blob/main/docs/contribute.md).
 - Contribute to the [documentation](https://github.com/Microsoft/vstest-docs).
 
+**IMPORTANT: We only accept PRs for which exists a previously discussed and approved issue.**
```

Comment:
move after Contributing so it's the first thing?

Reply:
ok